src/skyparams.h

```diff
@@ -38,6 +38,16 @@ struct SkyboxParams
 	s16 fog_distance { -1 };
 	float fog_start { -1.0f };
 	video::SColor fog_color { 0 }; // override, only used if alpha > 0
+
+	// Check if this is a textured skybox and whether transparency is used
```

Comment:
if the second half of the comment describes a different function it should be next to that function